### ISSUE TITLE
Finish drafting spina bifida and CWVV FAQ pages

### ISSUE THREAD

**Issue by laurwill** [department-of-veterans-affairs/va.gov-team#66683]
To finish the family member benefits batch in the community care migration project, we need to finish drafting the 2 planned FAQ pages about spina bifida and CWVV (children of women Vietnam Veterans) programs.

To do:

- [x] Review in process draft, finish writing the page, and call out remaining SME questions (Derek)
- [ ] Do final consistency review of all pages in batch and collect all SME questions (Laura)
- [ ] Send to SMEs (Danielle)
- [ ] Copyedit and publish (Megan)

**Comment by laurwill** [comment]
Hi @d-beyer , here's a ticket for the community care migration work we discussed last week. Let me know if you want to meet again to discuss as you're digging into this. 

cc @strelichl

**Comment by d-beyer** [comment]
Hi @laurwill -- I've reviewed the drafts, made changes and suggestions, and left comments and questions. Also, in case it's helpful, I've written up the questions for easier reference / sharing below. Let me know if you have any questions or there's anything else I can do!

## Health care for Children of Women Vietnam Veterans
### For SMEs
- It says the programs is for "covered birth defects". How does the user know what kind of birth defects are "covered"? Is there a list?
- It says the program pays for 100% of the "allowable amount". How is that determined? Is it listed somewhere?
- The draft refers to a program guide with more information. Is that available online? Should we refer to it?
- Is "allowable amount" a technical term? Could we rephrase? (Perhaps "the amount allowed by VA" or something to indicate who is doing the "allowing".)
- It says that the user can get pre-approval for certain kinds of care, including "durable medical equipment". By that do we mean reusable (and not disposable) equipment?
- The draft refers to "travel at the General Administrative Services rate". Where would you refer someone if they asked for that rate?
- That clause also says that the travel must be "other than mileage for local travel". What does that mean?
- The draft says that "Your provider should submit the preauthorization request for you". What should the user do if the provider doesn't do that?
### For Laura / Danielle
- The CTA links listed are "Apply for benefits for birth defects" and "Learn about filing claims for care". I wasn't able to locate live or draft pages that correspond to these. Where are we pointing?
- I wasn't able to find any content live or in draft about how to enroll in the program so I added a section. Does that fit with the purpose of this page? (I'm still getting a sense of what information we put where.)
- "Homemaker services" is listed as an approved type of care. Is there a way we typically refer to that kind of work, or do you have alternative suggestions?
## Getting care through the Spina Bifida Health Care Benefits Program
### For SMEs
- The draft refers to a program guide with more information. Is that available online? Should we refer to it?
- The draft says under services that aren't covered: "Services, procedures, or supplies that you don't have a legal obligation to pay (such as services you get at a health fair)". Is this saying that the program won't pay for services that don't have a cost, or aren't supposed to have a cost? Can we clarify this?
- It says that the user can get pre-approval for certain kinds of care, including "durable medical equipment". By that do we mean reusable (and not disposable) equipment?
- The draft says that the user can choose any provider that is approved by CMS, Tricare, the Joint Commission, or the state they live in. Are there places we can point the user to find these providers?
	- What is meant by "the state you live in"? Is there a consistent organization within states that makes this determination?
## How to file a claim for care for Spina Bifida or other birth defects
### For Laura/Danielle
- This page is for filing a claim, but not for enrollment in either program. The CTAs on the other two pages suggest that there may need to be a page for enrollment.
- We say "You can file a claim to ask us to pay you back (called 'reimbursement')." Is this a typical way to introduce a necessary technical term?
- This page provides a long list of supporting documents for the user to refer to. Are there other examples of how we would present/organize this kind of information?

**Comment by laurwill** [comment]
Hi @d-beyer ! Thanks so much for moving these drafts along — I've left you a bunch of comments and tracked edits in the Sharepoint document. Could you review and make some updates today before our meeting with Danielle and Mikki?

During that meeting, we can discuss how these new pages relate to the existing page about birth defects in the disability hub, and then adjust as needed.

**Comment by d-beyer** [comment]
Hey @laurwill -- I resolved comments and suggestions, made edits, and left one or two more notes. Let me know if you have any questions or what next steps you have for me!

**Comment by laurwill** [comment]
Hi @DanielleThierryUSDSVA , I just realized I never posted this for you! Sorry about that. Here's the document we looked at briefly with the spina bifida and CWVV pages in the health care benefit hub: https://dvagov-my.sharepoint.com/:w:/g/personal/danielle_thierry_va_gov/EW4S6Youj-pBlrgPM5faeR8BugNFKhAWtgi9rXW50cMSsg?e=ALoYg4 

The R&S page about filing a claim needs a bit more work, but wanted you to have these page drafts for now. 

And @d-beyer I'll follow up with you when we're back from break about finishing up the R&S "file a claim" page. Thanks for all your thorough work on these!
